src/no/runsafe/clans/database/ClanRepository.java

```diff
@@ -22,7 +25,7 @@ public ClanRepository(IDatabase database)
 		for (IRow row : database.query("SELECT `clanID`, `leader`, `motd`, `clanKills`, `clanDeaths`, `dergonKills` FROM `clans`"))
 		{
 			String clanName = row.String("clanID");
-			Clan clan = new Clan(clanName, row.String("leader"), row.String("motd"));
+			Clan clan = new Clan(clanName, playerProvider.getPlayer(UUID.fromString(row.String("leader"))), row.String("motd"));
```

Comment:
Didn't you make it so we could getPlayer using the uuid straight on the database row?